client/app/scripts/superdesk-highlights/highlights.js

```diff
@@ -165,77 +165,49 @@
     function HighlightsTitleDirective(highlightsService, $timeout) {
         return {
             scope: {
-                item: '=item',
-                orientation: '=?'
+                item: '=item'
             },
             templateUrl: 'scripts/superdesk-highlights/views/highlights_title_directive.html',
             // todo(petr): refactor to use popover-template once angular-bootstrap 0.13 is out
             link: function(scope, elem) {
-                var unmarkBox = elem.find('.unmark').hide(),
-                    icon = elem.find('i'),
-                    isOpen,
-                    closeTimeout;
 
-                scope.orientation = scope.orientation || 'right';
+                /*
+                 * Set 'open' class on dropdown menu element
+                 * @param {string} isOpen
```

Comment:
This docstring is a bit misleading, since the method itself does not set any CSS classes, it just sets/clears the `open` flag on the model (it's the template that later uses this flag). This also means that `isOpen` is not a string but a boolean.
The method does also not _set_ a value, it _toggles_ it.